operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -289,8 +359,7 @@
                 "mode": "absolute",
                 "steps": [
                   {
-                    "color": "green",
-                    "value": null
```

Comment:
hmm ... assumed the export did this .... I suppose it is OK

let's try to verify that @ramessesii2 after this merges

Reply:
yes, that's what I got when exported.